lib/blogs_api/post.ex

```diff
@@ -0,0 +1,25 @@
+defmodule BlogsApi.Post do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias BlogsApi.User
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  @foreign_key_type Ecto.UUID
+
+  schema "posts" do
+    field :title, :string
+    field :content, :string
+    belongs_to(:user, User)
+    timestamps()
+  end
+
+  @required_params ~w(title content user_id)a
+  #@spec changeset(:invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}) ::
```

Comment:
Se não for usar typespec aqui, poderia remover esse comentário 😄

Reply:
Bom ponto, Leandrão! Corrigido.
Obrigada por observar! :)